src/constants/request.ts

```diff
@@ -112,7 +112,7 @@ export const URLs = {
     get: '/quizzes',
     getById: '/quizzes/:id',
     add: '/quizzes',
-    patch: '/quizzes',
+    patch: '/quizzes/:id',
     delete: '/quizzes'
```

Comment:
```suggestion
    delete: '/quizzes/:id'
```
<img width="1438" alt="Screenshot 2025-02-07 at 13 43 39" src="https://github.com/user-attachments/assets/3bda7dac-fa06-4501-9fc9-619fd2d0a822" />

Reply:
I know this is not relevant to your pull request, but please fix this as well.